pkg/cmd/completion.go

```diff
@@ -20,10 +35,28 @@ func newCompletionCmd() *completionCmd {
 		Short: "Generate bash and zsh completion scripts",
 		Args:  validators.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
+			userOS := runtime.GOOS
+			if userOS == "windows" {
+				fmt.Println("Warning: autocompletion for Windows is currently not supported and may not work.")
+				// Give them instructions for linux because Windows _might_ be using the linux fs under the hood
```

Comment:
I think this is fine, but just to clarify: with WSL (Windows Subsystem for Linux), it's the Linux executable that is being run, and `runtime.GOOS` would (correctly) return `"linux"`, and the shell would actually be `bash` or `zsh` or some other Linux shell.